integration_tests/src/main/python/arithmetic_ops_test.py

```diff
@@ -500,3 +501,32 @@ def test_greatest(data_gen):
             lambda spark : gen_df(spark, gen).select(
                 f.greatest(*command_args)), conf=allow_negative_scale_of_decimal_conf)
 
+def _assert_div_by_zero(session_func, test_div_func, conf):
```

Comment:
Is it worth making something like this generic in asserts? I could see us wanting to use it in other situations.

Reply:
will do